# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/vm/Xdr.h
# Commit: f505b8fb8f8b
# Full Hash: f505b8fb8f8b7a8784be192ce6a36dc2278095cb
# Author: Tooru Fujisawa <arai_a@mac.com>
# Date: 2020-11-14 09:46:25
# Regressor Bug: 1675241
# File Overlap Count: 1
# Description:
#   Bug 1675241 - Part 8: Use TaggedParserAtomIndex in ScriptStencil.functionAtom. r=tcampbell
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D96915
# ==============================================================================

diff -r 3749d6d2325d -r f505b8fb8f8b js/src/vm/Xdr.h
--- a/js/src/vm/Xdr.h	Sat Nov 14 00:03:00 2020 +0000
+++ b/js/src/vm/Xdr.h	Sat Nov 14 00:03:07 2020 +0000
@@ -765,6 +765,10 @@
                         const frontend::ParserAtom** atomp);
 
 template <XDRMode mode>
+XDRResult XDRTaggedParserAtomIndex(
+    XDRState<mode>* xdr, frontend::TaggedParserAtomIndex* taggedIndex);
+
+template <XDRMode mode>
 XDRResult XDRParserAtomDataAt(XDRState<mode>* xdr,
                               const frontend::ParserAtom** atomp,
                               frontend::ParserAtomIndex index);
